src/languages/spark/spark.formatter.ts

```diff
@@ -144,7 +144,10 @@ export default class SparkFormatter extends Formatter {
       reservedFunctionNames: functions,
       openParens: ['(', '['],
       closeParens: [')', ']'],
-      stringTypes: [{ quote: "''", prefixes: ['X'] }],
+      stringTypes: [
+        { quote: "''", prefixes: ['r'] },
+        { quote: '""', prefixes: ['r'] },
+      ],
```

Comment:
I see you removed the hex-strings `X. These are still supported by Spark.

I updated the docs here as well: https://github.com/sql-formatter-org/sql-formatter/wiki/Strings

Reply:
missed those in the binary section